app/templates/view_suppliers.html

```diff
@@ -29,19 +29,18 @@
         empty_message="No suppliers were found",
         field_headings=[
           "Name",
+          summary.hidden_field_heading("Edit name"),
           summary.hidden_field_heading("Users"),
           summary.hidden_field_heading("Services"),
         ],
         field_headings_visible=True)
       %}
-      {% call summary.row() %}
-
-      {{ summary.field_name(item.name) }}
-
-      {{ summary.service_link("Users", url_for(".find_supplier_users", supplier_id=item.id)) }}
-      {{ summary.service_link("Services", url_for(".find_supplier_services", supplier_id=item.id)) }}
-
-      {% endcall %}
+        {% call summary.row() %}
+          {{ summary.field_name(item.name) }}
+          {{ summary.edit_link("Change name", url_for(".edit_supplier_name", supplier_id=item.id)) }}
```

Comment:
It would be good to only show the `Change name` to `admin` users - at the moment `admin-ccs-category` users can also see this page.  I think they should be able to browse it - and click through to see the list of users & services - but not to make any changes.